Engine/AlgorithmManager.cs

```diff
@@ -514,7 +517,7 @@ public void Run(AlgorithmNodePacket job, IAlgorithm algorithm, ISynchronizer syn
                         Log.Debug($"AlgorithmManager.Run(): {algorithm.Time}: Applying Split for {split.Symbol}");
 
                         Security security = null;
-                        if (_liveMode && algorithm.Securities.TryGetValue(split.Symbol, out security))
+                        if (_liveMode & algorithm.Securities.TryGetValue(split.Symbol, out security))
```

Comment:
Same here. `&` is bitwise comparison which might work in this particular case but shouldn't be relied on.

Reply:
When using `&` with booleans, the net result will be the same, but the more important aspect of `&&` is it's short-circuiting functionality, so if `_liveMode` is `false`, then `algorithm.Securities.TryGetValue` is never invoked

Reply:
In the last commit, I flipped the order:
```csharp
if (algorithm.Securities.TryGetValue(split.Symbol, out security && _liveMode)
```
The changed were made to precisely get `algorithm.Securities.TryGetValue` invoked.